### PR TITLE
Adding buildvm snapshot email to buildvm-maint

### PR DESCRIPTION
**Pull request by adammhaile**
@jupierce - We can certainly do something other than / in addition to email. This was just a very quick first pass to layout what I was thinking of capturing.

### CODE REVIEW

**Comment by adammhaile** [comment]
Haha, good idea. Doubt we'll really get emails that often... but would also
be a good sanity check, letting us know that something has changed we
weren't expecting.
Maybe even store to /home/jenkins the actual diff (still send the whole
thing in email) so that we could login and see what the diff was, should we
care.

**Comment by adammhaile** [comment]
@jupierce updated to include diff suggestion

**Comment by adammhaile** [comment]
@jupierce The requested changes have already been addressed here

File: jobs/build/buildvm-maint/scripts/gen_snapshot.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+echo ""
+echo "INSTALLED RUBY GEMS"
+echo "==================="
+gem list --local
+
+echo ""
+echo "DOCKER INFO"
+echo "==========="
+docker info
```
Comment (jupierce) [inline jobs/build/buildvm-maint/scripts/gen_snapshot.sh]: docker info will change constantly since it lists freespace. Incompatible with a diff model.

**Comment by adammhaile** [comment]
@jupierce Removed docker info

**Pull request merged**